packages/concerto-core/test/introspect/scalardeclaration.js

```diff
@@ -102,11 +102,9 @@ describe('ScalarDeclaration', () => {
             const modelFiles = introspectUtils.loadModelFiles([modelFileName], modelManager);
             modelManager.addModelFiles(modelFiles);
         });
-        it('should throw an error', () => {
+        it('should return null', () => {
             const testClass = modelManager.getType('com.testing.SSN');
-            (() => {
-                testClass.getSuperType();
-            }).should.throw('Scalars do not have super types.');
+            should.equal(testClass.getSuperType(), null);
```

Comment:
Why return null here?

Reply:
Good question. I'm adding a test to increase coverage which reflects the current implementation.

 This is one of the now deprecated functions. It should go away in the next major release, IMO